sklearn/manifold/_debugvars.h

```diff
@@ -0,0 +1,3 @@
+#define DEBUGFLAG 0
```

Comment:
Is it really necessary to introduce a new file just for this? As `_debugvars.h` is only included in `_barnes_hut_tsne.pyx` I would rather put a an inline definition of that constant. AFAIK there is no Cython keyword to default a `const` variable but maybe a `cdef enum` would do. It needs to be bench to make sure that it does not cause a performance regression but I think it should be fine.